### PR TITLE
Fix Fallout Forum ID

### PR DESCRIPTION
**Pull request by vforge**
/cc @justnpT @tomasznapieralski @kvas-damian

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java [hunk 1]
```diff
@@ -73,7 +73,7 @@
   private WebElement googlePlayAppLink;
 
   private static final String PATH = "d/f/%s";
-  private static final String DEFAULT_FORUM_ID = "203236";
+  private static final String DEFAULT_FORUM_ID = "3035"; // ForumID on Fallout
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/discussions/PostsListPage.java]: MINOR Move this trailing comment on the previous empty line. [rule](http://sonar-s1/coding_rules#rule_key=squid%3ATrailingCommentCheck)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by tomasznapieralski** [comment]
LGTM

**Comment by vforge** [comment]
Turned out it wasn't best experience - we swapped one failing tests with others.

**Pull request merged**